### PR TITLE
Add draft Readme.MD

### PR DESCRIPTION
**Pull request by nick863**
# Description

Add draft Readme.MD to allow uploading package to pypi.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [x] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow-evals/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-tools)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are prebuilt promptflow pipelines that are designed to measure the quality of the outputs from large language models.
+The package includes
+ - Chat evaluator
```
Comment (wangchao1230) [inline src/promptflow-evals/README.md:9]: may have some one line description explain what these evaluator does?

**Review by luigiw — APPROVED** [review]

File: src/promptflow-evals/README.md (hunk 1 above)
Comment (nick863) [reply src/promptflow-evals/README.md]: I have added more context to evaluators.

File: src/promptflow-evals/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-tools)](https://pypi.org/project/promptflow-evals/)
```
Comment (wangchao1230) [inline src/promptflow-evals/README.md:3]: This link is incorrect pointing to tools

Comment (nick863) [reply src/promptflow-evals/README.md:3]: Fixed (however, the evals are not on pypi yet)

File: src/promptflow-evals/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-tools)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are prebuilt promptflow pipelines that are designed to measure the quality of the outputs from large language models.
+The package includes
```
[the remark this answers is not in the document]
Comment (nick863) [reply src/promptflow-evals/README.md]: Fixed.

File: src/promptflow-evals/README.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-evals)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are prebuilt promptflow pipelines that are designed to measure the quality of the outputs from language models.
```
Comment (singankit) [inline src/promptflow-evals/README.md:7]: These can be custom in addition to prebuilt

Comment (nick863) [reply src/promptflow-evals/README.md]: Fixed.

**Review by singankit — DISMISSED** [review]

**Review by nagkumar91 — DISMISSED** [review]

File: src/promptflow-evals/README.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-evals)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are custom or prebuilt promptflow pipelines that are designed to measure the quality of the outputs from language models.
+
+## Usage
+Users can create run evaluators on the local machine as shown in the example below:
```
Comment (ninghu) [inline src/promptflow-evals/README.md:10]: create or run?

File: src/promptflow-evals/README.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    # Initialize Project Scope
+    project_scope = {
+        "subscription_id": "e0fd569c-e34a-4249-8c24-e8d723c7f054",
+        "resource_group_name": "rg-test",
+        "project_name": "project-test",
+    }
+
+    violence_eval = ViolenceEvaluator(project_scope)
+    violence_score = violence_eval(question="What is the capital of France?", answer="Paris.")
+    pprint(violence_score)
```
Comment (ninghu) [inline src/promptflow-evals/README.md:53]: these two methods are not being referenced anywhere, which can be removed.

Comment (nick863) [reply src/promptflow-evals/README.md]: Removed

File: src/promptflow-evals/README.md (hunk 5 above)
Comment (nick863) [reply src/promptflow-evals/README.md]: Fixed

File: src/promptflow-evals/README.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
+# Prompt flow evaluators
+
+[![Python package](https://img.shields.io/pypi/v/promptflow-evals)](https://pypi.org/project/promptflow-evals/)
+[![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
+
+## Introduction
+Evaluators are custom or prebuilt promptflow pipelines that are designed to measure the quality of the outputs from language models.
```
[the remark this answers is not in the document]
Comment (nick863) [reply src/promptflow-evals/README.md]: Fixed.

**Review by singankit — APPROVED** [review]

**Review by ninghu — APPROVED** [review]

**Pull request merged**